tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
That is not the type of comment that is given here. These comments are specifications:

````
/**
 * It should retrieve a media entity by ID
 */
````

Would be valid. This is a *policy* which helps write better tests and is the reason I have adopted it here.